airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/abstract_stream.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import logging
+from abc import ABC, abstractmethod
+from typing import Any, Iterable, List, Mapping, Optional, Tuple, Union
+
+from airbyte_cdk.sources.streams.concurrent.partitions.record import Record
+from deprecated.classic import deprecated
+
+PrimaryKey = Union[str, List[str]]
```

Comment:
`List[str]` is intended to support the compound primary key use case (and not nested keys) correct?

Reply:
That's correct